web/src/main/webapp/dataViews/schemaView.ts

```diff
@@ -38,16 +38,34 @@ export class SchemaView extends RemoteTableObjectView {
                 public schema: Schema,
                 private rowCount: number) {
         super(remoteObjectId, page);
-        this.topLevel = document.createElement("div");
-
         this.topLevel = document.createElement("div");
         let subMenu = new SubMenu([
             {text: "Selected columns",
                 action: () => {this.showTable();},
                 help: "Show the data using a tabular view containing the selected columns."
             }
         ]);
-        let menu = new TopMenu([{text: "View", help: "Change the way the data is displayed.", subMenu}]);
+        let selectMenu = new SubMenu([
+            {
+                text: "By Name",
+                action: () => {nameDialog.show();},
+                help: "Select Columns by name."
+            },
+            {
+                text: "By Type",
+                action: () => {typeDialog.show();},
+                help: "Select Columns by type."
+            },
+            {
+                text: "By Allows Missing",
+                action: () => {missingDialog.show();},
```

Comment:
Where is missingDialog?

Reply:
It is right after nameDialog and typeDialog. Starts at line 112.